src/gcp_scanner/crawl.py

```diff
@@ -186,9 +182,7 @@ def get_compute_disks_names(
     while request is not None:
       response = request.execute()
       if response.get("items", None) is not None:
-        for _, disks_scoped_list in response["items"].items():
-          for disk in disks_scoped_list.get("disks", []):
-            disk_names_list.append(disk)
+        disk_names_list = [disk for _, disks_scoped_list in response["items"].items() for disk in disks_scoped_list.get("disks", [])]
```

Comment:
Split to multiple lines for readability.

Reply:
Hi @ZetaTwo , I have resolved this issue in the latest PR.